runtime/gc_base/ObjectCheck.cpp

```diff
@@ -50,7 +50,7 @@ isValidClass(J9JavaVM *javaVM, J9Class *ptr, UDATA flags) {
 	}
 
 	/* check alignment */
-	if (0 != ((UDATA)ptr & (sizeof(UDATA) - 1))) {
+	if (0 != ((uintptr_t)ptr & (sizeof(uintptr_t) - 1))) {
```

Comment:
while we can keep this change minimalistic (formatting only), we should probably use a proper const/variable rather then sizeof(uintptr_t)

we objectAlignmentInBytes available at a few places 

although I'm a bit confused - ptr is pointer to RAM class, should we not have a strong check for 256 byte (for what we have J9_REQUIRED_CLASS_ALIGNMENT)!?

Reply:
Yes, it caught my eye too, we need to check correctness as well is this code in use or obsolete. I added this to my todo list. I think it is better to keep this PR formatting only.
Regarding `sizeof(uintptr_t)` I thought about this too. Do we have better declaration?